src/xdocs/config_naming.xml

```diff
@@ -1145,7 +1145,25 @@ class MyClass {
       <subsection name="Description">
         <p>
           Checks that method and <code>catch</code> parameter names conform to a format specified
-          by the format property.
+          by the format property. By using <code>scope</code> and <code>excludeScope</code> properties
+          it is possible to specify different formats for methods at different visibility levels.
+          For example, the following configuration checks that the parameters always start with two
+          lowercase characters and, in addition, that public method parameters cannot be one character
+          long:
+          <pre>
```

Comment:
Examples must be in `<source>` tag, not `<pre>` to be validated by our unit tests.